cbr-service-profile/README.md

```diff
@@ -1,47 +1,47 @@
-# CBR Rule Profile
-
-Accepts a list of VPC crns / service references to create CBR zones and a list of target services, to create the rule matching these profiles.  It supports to target the service using name, account id, tags, resource group.
+# CBR service profile
+
+This profile creates a list of CBR zones and rules. Accepts a list of VPC CRNs / service references to create CBR zones and a list of target services to create the rule matching these profiles. It supports to target the service using name, account id, tags, resource group.
```

Comment:
I'd use the term "module" here instead of "profile"

Reply:
renamed from "profile" to "module"